### DISCUSSION TITLE
GHA docs say `macos-latest` is `macos-14` (M1) but it is instead `macos-12` (Intel)

### DISCUSSION THREAD

**Discussion by cclauss** [community/community#116911] [Actions]
### Select Topic Area

Bug

### Body

* actions/runner-images#9640

**Comment by cyqsimon** [comment]
Ran gob smack into this while working on [bandwhich](https://github.com/imsnif/bandwhich). Had to [find out what exactly was happening experimentally](https://github.com/imsnif/bandwhich/actions/runs/8688298225/job/23823640441#step:5:17), wasting several hours in the process 😮‍💨.

Seriously, wrong documentation is worse than no documentation.
